Gold layer notebook of usecase 1

MeghanaTData opened the pull request:
Added Gold Layer aggregation for IMDB project

- Aggregated data by genre and year
- Calculated average rating and total number of votes
- Filtered out entries with missing genres
- Extracted year from startYear column

Commit:
Gold layer notebook of usecase 1

Aggregated no of votes and avg rating of movies per genre per year

vamsi-simbus commented on src/Gold Notebook Usecase 1/Gold layer IMDB.ipynb:
```diff
@@ -0,0 +1,146 @@
+{
```
Suggestion to maintain folder structure as Gold_Notebook

vamsi-simbus commented on src/Gold Notebook Usecase 1/Gold layer IMDB.ipynb:
```diff
@@ -0,0 +1,146 @@
+   "source": [
+    "# Creating the Title_df DataFrame from the silver layer table\n",
+    "silver_df = spark.table('az_adb_simbus_training.imdb_project_schema.silver_usecase1_table')\n",
+    "silver_df.limit(5).display()"
```
please comment display statements

Commit:
Made Suggested Changes in Gold layer table

1. Comment out Display command
2. Selected only two decimal from the Average rating column
